core-web/libs/ui/src/lib/components/dot-drop-zone/dot-drop-zone.component.ts

```diff
@@ -0,0 +1,135 @@
+import { CommonModule } from '@angular/common';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    EventEmitter,
+    HostBinding,
+    HostListener,
+    Input,
+    Output
+} from '@angular/core';
+
+export enum DropZoneError {
+    INVALID_FILE = 'INVALID_FILE',
+    MULTIFILE_ERROR = 'MULTIFILE_ERROR'
+}
+
+@Component({
+    selector: 'dot-drop-zone',
+    standalone: true,
+    imports: [CommonModule],
+    templateUrl: './dot-drop-zone.component.html',
+    styleUrls: ['./dot-drop-zone.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DotDropZoneComponent {
+    @Output() fileDrop = new EventEmitter<File>();
+    @Output() dropZoneError = new EventEmitter<DropZoneError>();
+    @Output() dragStart = new EventEmitter<boolean>();
+    @Output() dragStop = new EventEmitter<boolean>();
+
+    @Input() allowedExtensions: string[] = [];
+    @Input() set allowedMimeTypes(mineTypes: string[]) {
+        this._allowedMimeTypes = mineTypes.map((type) => {
+            // Remove the wildcard character
+            return type.toLowerCase().replace(/\*/g, '');
+        });
+    }
```

Comment:
Can we group this in an `accept` or `allowed` and it can take either mime or extensions?

Reply:
Yes sir!